sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.models.ts

```diff
@@ -127,6 +129,21 @@ export const FULL_TIME_DISBURSEMENT_FEEDBACK_ERRORS = [
   "EDU-00112",
 ];
 
+/**
+ * Represents students Disability status both from application submitted
+ * and the student profile disability status verification.
+ */
+export interface DisabilityDetails {
+  /**
+   * WorkflowData from the assessment.
+   */
+  workflowData: WorkflowData;
```

Comment:
It doesn't need to have all the workflow data. Only pdppdStatus is sufficient here.